### PR TITLE
First pass at Variety 2020 and VIP

### PR DESCRIPTION
**Pull request by emeaguiar**
This is a first pass at the Variety 2020 and VIP tokens needed to build a few components in the site, mainly basic fonts, footer, and article header components.

### CODE REVIEW

File: packages/larva-tokens/src/brands/variety.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+			"value": "{!GRAPHIK_COND}",
+			"type": "font-family",
+			"category": "font-family"
+		},
+		"FONT_FAMILY_BODY": {
+			"value": "{!IBM_PLEX_SERIF}",
+			"type": "font-family",
+			"category": "font-family"
+		},
+		"FONT_FAMILY_BASIC": {
```
Comment (laras126) [inline packages/larva-tokens/src/brands/variety.json]: Anything that is a webfont needs to have a fallback - basic is intended to be a websafe font like Helvetica. If it is a webfont, it should be added as FONT_FAMILY_BASIC_FANCY with a fallback, FONT_FAMILY_BASIC.

File: packages/larva-tokens/src/brands/variety.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+			"value": "{!PARA_SUPREME}",
+			"type": "font-family",
+			"category": "font-family"
+		},
+		"FONT_FAMILY_SECONDARY_FANCY": {
+			"value": "{!IBM_PLEX_SANS}",
+			"type": "font-family",
+			"category": "font-family"
+		},
+		"FONT_FAMILY_ACCENT": {
```
Comment (laras126) [inline packages/larva-tokens/src/brands/variety.json]: Same note as below about websafe font version.

File: packages/larva-tokens/src/brands/variety.json (hunk 1 above)
Comment (laras126) [reply packages/larva-tokens/src/brands/variety.json]: Also - can this be FONT_FAMILY_MONO, as above? The tokens between Variety and Variety VIP should be as similar as possible

File: packages/larva-tokens/src/brands/variety-vip.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+			"value": "{!IBM_PLEX_SANS}",
+			"type": "font-family",
+			"category": "font-family"
+		},
+		"FONT_FAMILY_BODY": {
+			"value": "{!TIMES_NEW_ROMAN}",
+			"type": "font-family",
+			"category": "font-family"
+		},
+		"FONT_FAMILY_MONO": {
```
Comment (laras126) [inline packages/larva-tokens/src/brands/variety-vip.json]: What is the reason for having IBM_PLEX_MONO as FONT_FAMILY_MONO here vs. FONT_FAMILY_BASIC below? Can they be the same?

File: packages/larva-tokens/src/brands/variety.json (hunk 1 above)
Comment (emeaguiar) [reply packages/larva-tokens/src/brands/variety.json]: This is an overlook in my part `FONT_FAMILY_MONO` should be `FONT_FAMILY_ACCENT` for vip, which I'll update. Thanks for the catch!

These names are taken from the invision styleguides, where Variety and VIP share a naming convention, but differ in values.

File: packages/larva-tokens/src/brands/variety-vip.json (hunk 3 above)
Comment (emeaguiar) [reply packages/larva-tokens/src/brands/variety-vip.json]: As I mentioned above, this should be `FONT_FAMILY_ACCENT` for vip and `FONT_FAMILY_BASIC` for variety based on the styleguides, I will update.

File: packages/larva-tokens/src/brands/variety.json (hunk 2 above)
Comment (emeaguiar) [reply packages/larva-tokens/src/brands/variety.json]: I've updated with fallbacks

File: packages/larva-tokens/build/variety.custom-properties.css [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  --background-color-grey-dark: rgb(106, 106, 106);
+  --background-color-grey-darkest: rgb(50, 50, 50);
+  --background-color-black: rgb(0, 0, 0);
+  --color-brand-secondary: rgb(65, 126, 244);
+  --font-family-basic-fancy: 'IBM Plex Mono', monospace;
+  --color-brand-primary: rgb(238, 232, 9);
+  --font-family-body-fancy: 'IBM Plex Serif', serif;
+  --border-color-brand-primary: rgb(65, 126, 244);
+  --background-color-brand-primary: rgb(238, 232, 9);
+  --background-color-brand-accent-a-100-b: rgb(26, 40, 47);
```
Comment (laras126) [inline packages/larva-tokens/build/variety.custom-properties.css]: Where is this coming from? What does `a-100-b` represent?

For colors, I'm also guessing there will be several for VY that are not in the existing tokens. Added some guidelines for that here: https://github.com/penske-media-corp/pmc-larva/wiki/CSS-Class-Generators#basic-utility-generator

**Comment by emeaguiar** [comment]
@laras126 I made some updates and remove the variants of the colors to put them in the theme itself, make sense to me since this styleguide has a lot of colors. So let me know if something else caught our eyes.

File: packages/larva-tokens/src/brands/variety-vip.json [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+			"type": "color",
+			"category": "text-color"
+		},
+		"FONT_FAMILY_PRIMARY_FANCY": {
+			"value": "{!GRAPHIK_COND}",
+			"type": "font-family",
+			"category": "font-family"
+		},
+		"FONT_FAMILY_SECONDARY": {
+			"value": "{!BONNIE}",
```
Comment (laras126) [inline packages/larva-tokens/src/brands/variety-vip.json]: This should be a web safe font

**Comment by laras126** [comment]
@emeaguiar Almost there! 

I think the naming for the webfonts is confusing - `_FANCY` = webfont, (in retrospect _WEBFONT may be a better name...) and every font family should have a base version e.g. FONT_FAMILY_ACCENT set to a fallback font, and then the webfont assigned to `_FANCY`.

So, from what I see in the PR, there needs to be fallback for FONT_FAMILY_ACCENT, and it looks like a webfont is assigned to FONT_FAMILY_SECONDARY, when that should be the fallback font.

**Comment by emeaguiar** [comment]
Hey @laras126 , thanks for the explanation!

I have updated the fallback fonts for the secondary typeface and added a new accent fallback, let me know if there's anything else that needs to be done.

**Comment by emeaguiar** [comment]
Hey @laras126 just reminding you this is here, let me know if something else is needed

**Comment by mehigh** [comment]
@laras126 can you review in order to plan to get it merged in prior to the holidays season? Anything we can support to get this moving?

**Comment by mehigh** [comment]
@laras126 can we at least get a response on when we can expect this to get merged in?

**Comment by laras126** [comment]
@mehigh Looking now - sorry about the delay. Was OOO last week, and will merge this today.

**Pull request merged**